test/controllers/transport/UsePrintAndPostControllerSpec.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.transport
+
+import base.SpecBase
+import config.{ClaimAmounts, NavConstant}
+import controllers.actions.UnAuthed
+import models.{NormalMode, UserAnswers}
+import navigation.{FakeNavigator, Navigator}
+import org.mockito.ArgumentMatchers._
+import org.mockito.Mockito._
+import org.scalatest.OptionValues
+import org.scalatest.concurrent.{IntegrationPatience, ScalaFutures}
+import org.scalatestplus.mockito.MockitoSugar
+import pages.ClaimAmount
+import pages.transport.AirlineJobListPage
+import play.api.Application
+import play.api.inject.bind
+import play.api.mvc.Call
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+import repositories.SessionRepository
+
+import scala.concurrent.Future
+
+class UsePrintAndPostControllerSpec extends SpecBase with ScalaFutures with MockitoSugar with IntegrationPatience with OptionValues {
+
+  def onwardRoute: Call = Call("GET", "/foo")
+
+
+  lazy val UsePrintAndPostControllerRoute: String = routes.UsePrintAndPostController.onPageLoad().url
+
+  "UsePrintAndPostController Controller" must {
+
+    "return OK for a GET" in {
+
+      val application = applicationBuilder(userAnswers = Some(emptyUserAnswers)).build()
+
+      val request = FakeRequest(GET, UsePrintAndPostControllerRoute)
+
+      val result = route(application, request).value
+
+      status(result) mustEqual OK
```

Comment:
IT's up to you, but you could also test if the specific view is being called. Similarly to what I've done here: https://github.com/hmrc/employment-expenses-tax-relief-guidance-frontend/blob/main/test/controllers/UsePrintAndPostControllerSpec.scala